.github/workflows/validate-and-report.yml

```diff
@@ -191,22 +191,22 @@ jobs:
             make force_extract
             .venv/bin/python3 tools/analyze_calls.py
             git clean -fdx asm/
+      - name: Generate duplicates report
+        run: |
+          make force_symbols
+          make force_extract
+          cd tools/dups
+          cargo run --release -- --threshold .90 --output-file ../../gh-duplicates/duplicates.txt
```

Comment:
Since the function report relies on the output of running this, we need to shift the order. I'm not sure if doing so has dependent steps that need to be moved in here like checkout / rm -rf from Generate function report